libdebug/data/gdb_resume_event.py

```diff
@@ -0,0 +1,40 @@
+#
+# This file is part of libdebug Python library (https://github.com/libdebug/libdebug).
+# Copyright (c) 2024 Roberto Alessandro Bertolini. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from libdebug.debugger.internal_debugger import InternalDebugger
+
+
+class GdbResumeEvent:
+    """This class handles the actions needed to resume the debugging session, after returning from GDB."""
+
+    def __init__(
+        self: GdbResumeEvent,
+        internal_debugger: InternalDebugger,
+        lambda_function: callable[[], None],
```

Comment:
[] -> list[]

Reply:
Why? No?
`callable[[], None]` is the canonical way of typing a niladic function